### DISCUSSION TITLE
Salt Project Moved Repo To Broadcom

### DISCUSSION THREAD

**Discussion by br3nj1** [Security-Onion-Solutions/securityonion#14017] [2.4] [answered]
### Version

2.4.10

### Installation Method

Network installation on Ubuntu (unsupported)

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

32GB

### Storage for /

1TB

### Storage for /nsm

1TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

The Salt Project has migrated its package repositories from repo.saltproject.io to packages.broadcom.com.

Since repo.saltproject.io is no longer available, network installations are failing. 

Repository Migration: 
Deprecated Repo: repo.saltproject.io
New Repo: packages.broadcom.com

Recommendations:
Update Repository Configurations: Modify your system's package manager configurations to point to the new repository URLs.

References:
https://saltproject.io/blog/

[Upcoming bootstrap subdomain decommission](https://saltproject.io/blog/upcoming-bootstrap-decommission/)

[FAQs from Salt Project Repo Migration](https://saltproject.io/blog/post-migration-salt-project-faqs/)

Has anyone seen plans about Security Onion editing the install config? How can we proceed with network installations? 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
https://github.com/Security-Onion-Solutions/securityonion/issues/13898

**Reply by br3nj1** [reply]
Thanks. I searched but didn't find. Was stoked, thought I identified another curl issue. Almost sort of.

**Comment by NeoLife-iT** [comment]
it is mind boggling that there is no 301 redirect setup for this move form salt project.  where have common sense gone?

**Reply by TOoSmOotH** [reply]
I don't think planning was involved when they made this change. More like you must move by the end of the week and the the rug got pulled. I think there was something like 3 days notice and a couple of weeks before bootstrap got fixed.

**Comment by br3nj1** [comment]
How do you fix this within a debian network install? 

Would following this guide get so-setup woring?
https://docs.saltproject.io/salt/install-guide/en/latest/topics/install-by-operating-system/linux-deb.html 

Is the salt repo a variable in so-setup....Can we the change DNS address in one place?

**Reply by dougburks** [reply, marked as answer]
My previous response linked to the fix which will be included in our next release:
https://github.com/Security-Onion-Solutions/securityonion/pull/13900

You can try applying this fix manually. However, please keep in mind that network installations are not supported. From https://docs.securityonion.net/en/2.4/network-installation.html:
![image](https://github.com/user-attachments/assets/572bf063-bdd5-4438-bf90-53d4d44950e9)

If you want a supported installation, you will need to use our Security Onion ISO image.

**Reply by br3nj1** [reply]
I'm up and running, great to see an iso built on ubuntu. my AMD couldn't run CentOs versions. All set.
